ci/values.yaml

```diff
@@ -0,0 +1,141 @@
+default:
+  env:
+    - name: OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE
+      value: cumulative
+    - name: OTEL_SERVICE_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: "metadata.labels['app.kubernetes.io/component']"
+    - name: OTEL_K8S_NAMESPACE
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.namespace
+    - name: OTEL_K8S_NODE_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: spec.nodeName
+    - name: OTEL_K8S_POD_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.name
+    - name: OTEL_K8S_POD_UID
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.uid
+    - name: POD_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: status.podIP
+    - name: HOST_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    - name: OTEL_COLLECTOR_NAME
+      value: $(HOST_IP)
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://$(OTEL_COLLECTOR_NAME):4317
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: >-
+        service.name=$(OTEL_SERVICE_NAME),
+        service.instance.id=$(OTEL_K8S_POD_UID),
+        ervice.namespace=opentelemetry-demo,
+        k8s.namespace.name=$(OTEL_K8S_NAMESPACE),
+        k8s.node.name=$(OTEL_K8S_NODE_NAME),
+        k8s.pod.name=$(OTEL_K8S_POD_NAME),
+        deployment.environment=$(OTEL_K8S_NAMESPACE),
+        k8s.pod.ip=$(POD_IP)
```

Comment:
This needs to be in both the app and the collector pods for container tags & metrics to work correctly.

Reply:
Do you mean all the resource attributes ?. Could you please provide some context why is it essential for collector pods ?

Reply:
Because the collector pods are emitting metrics from their receivers, and those metrics are passing through the k8sattributesprocessor which can add (very important) attributes to those metrics. The processor won't be able to associate with the pod if the IP is not provide as a resource attribute.

Reply:
As discussed offline, we will add this in collector in a followup PR